kakao/src/main/kotlin/com/agoda/kakao/Views.kt

```diff
@@ -462,7 +462,7 @@ class KListView(builder: ViewBuilder.() -> Unit, itemTypeBuilder: KAdapterItemTy
  * @see KRecyclerItemTypeBuilder
  */
 class KRecyclerView(builder: ViewBuilder.() -> Unit, itemTypeBuilder: KRecyclerItemTypeBuilder.() -> Unit)
-    : RecyclerActions, BaseAssertions {
+    : RecyclerActions, RecyclerAssertions, BaseAssertions {
```

Comment:
If class starts implement `RecyclerAssertions`, there is no need in implementing `BaseAssertions` since `RecyclerAssertions` actually extend one.